crates/superposition_types/src/lib.rs

```diff
@@ -243,9 +244,59 @@ impl Display for RegexEnum {
     }
 }
 
-#[derive(Clone, Deserialize)]
+#[derive(Clone, Debug, Deserialize)]
+pub enum HeadersEnum {
+    ConfigVersion,
+}
+
+impl fmt::Display for HeadersEnum {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            HeadersEnum::ConfigVersion => write!(f, "Config-Version"),
+        }
+    }
+}
+
+#[derive(Clone, Debug, Deserialize)]
+pub enum HttpMethod {
+    Get,
+    Post,
+    Put,
+}
+
+#[derive(Clone, Debug, Deserialize)]
+pub struct Authorization {
+    pub key: String,
+    pub value: String,
+}
+
+#[derive(Clone, Debug, Deserialize)]
+pub struct Webhook {
+    pub url: String,
+    pub method: HttpMethod,
+    pub headers: Vec<HeadersEnum>,
+    pub authorization: Value,
```

Comment:
shouldn't the type here be `Authorization` ?
```suggestion
    pub authorization: Authorization,
```